src/main/java/uk/gov/hmcts/reform/sscs/service/FooterService.java

```diff
@@ -60,5 +61,23 @@ protected void addDocumentToCaseDataDocuments(SscsCaseData caseData, SscsDocumen
         caseData.setSscsDocument(documents);
     }
 
+    public void setHearingDateAsExcludeDate(Hearing hearing, SscsCaseData sscsCaseData) {
+        List<ExcludeDate> newExcludeDates = new ArrayList<>();
+        if (sscsCaseData.getAppeal().getHearingOptions().getExcludeDates() != null) {
+            newExcludeDates.addAll(sscsCaseData.getAppeal().getHearingOptions().getExcludeDates());
+        }
+
+        DateRange dateRange = DateRange.builder()
+                .start(getLocalDate(hearing.getValue().getHearingDate()))
+                .end(getLocalDate(hearing.getValue().getHearingDate()))
+                .build();
+        newExcludeDates.add(ExcludeDate.builder().value(dateRange).build());
+
+        sscsCaseData.getAppeal().getHearingOptions().setExcludeDates(newExcludeDates);
+    }
 
+    private static String getLocalDate(String dateStr) {
+        LocalDate localDate = LocalDate.parse(dateStr, DateTimeFormatter.ofPattern(DATE_FORMAT_YYYYMMDD));
+        return localDate.toString();
+    }
```

Comment:
No need for a local variable.  Just do this:
return LocalDate.parse(dateStr, DateTimeFormatter.ofPattern(DATE_FORMAT_YYYYMMDD)).toString();